packages/tools/devtools/devtools-browser-extension/EXTENSION_CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Fluid Framework Developer Tools
 
+# Unpublished
+
+# 0.1.1
```

Comment:
Nit (Markdown best practices call for a newline between headings and content, and also surrounding lists):
```suggestion

# 0.1.1
```